.github/workflows/labeler.yml

```diff
@@ -1,16 +1,12 @@
-# Workflow to associate labels automatically
-name: labeler
-# Trigger the workflow on pull request events
+name: "Pull Request Labeler"
 on: [pull_request]
+
 jobs:
-  label:
-    runs-on: ubuntu-20.04
+  triage:
+    runs-on: ubuntu-latest
```

Comment:
I think it's better to pin the version, so `ubuntu-20.04` in this case. So we have control on it.